rasa/model.py

```diff
@@ -346,12 +336,12 @@ def merge_model(source: Text, target: Text) -> bool:
         shutil.move(source, target)
         return True
     except Exception as e:
-        logging.debug(e)
+        logging.debug("Could not merge model: {}".format(e))
         return False
 
 
 def should_retrain(new_fingerprint: Fingerprint, old_model: Text, train_path: Text):
```

Comment:
A namedtuple would be even better
```suggestion
def should_retrain(new_fingerprint: Fingerprint, old_model: Text, train_path: Text) -> Tuple[bool, bool, bool]: 
```

Reply:
let's do a namedtuple or even a class. then you also just have to pass one parameter around instead of 4.

Reply:
You mean 3?

Reply:
I'm halfway though making it a `namedtuple`, and it's seeming more complicated than it's worth it.

Reply:
Actually 4 cause you also had the `force` in the `train` module